subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1593,34 +1599,35 @@ int nrf_cloud_pgps_begin_update(void)
 	/* assume cache is no longer valid */
 	discard_prediction_buffer();
 
+	index.loading_count = 0;
+	index.store_block = npgps_alloc_block();
+	if (index.store_block == NO_BLOCK) {
+		LOG_ERR("No free flash space!");
+		return -ENOMEM;
```

Comment:
@plskeggs Looks like there should be a npgps_download_unlock() call here?

Reply:
pete is out this week.
yes it looks like an unlock is needed here.
curious that this block of code was just relocated but the unlock line was removed.
i have an open PR, i'll add a commit.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/13775/commits
it belongs to that PR